CHANGELOG.md

```diff
@@ -1,5 +1,73 @@
 ## Unreleased
 
+<<<<<<< HEAD
```

Comment:
this one looks like it still has the conflict?

Reply:
woops, thanks!